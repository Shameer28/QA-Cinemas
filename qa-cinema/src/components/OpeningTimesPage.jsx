const OpeningTimesPage = () => {
    return (
        <div>

        </div>
    );
}

export default OpeningTimesPage;