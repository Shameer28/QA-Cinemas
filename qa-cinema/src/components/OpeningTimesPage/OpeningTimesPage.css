tr{
text-align: center;
}
#o1{
    text-align: center;
}
