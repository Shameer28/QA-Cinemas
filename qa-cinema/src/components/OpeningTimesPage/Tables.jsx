import Table from 'react-bootstrap/Table';
import './OpeningTimesPage.css';

const Tables = () => {
    return (
        <div>
            <Table striped bordered hover size="sm" variant="dark" style={{ backgroundColor: "#212529", color: "white", border: "solid", borderColor: "black" }}>
                <thead>
                    <tr>
                        <th><u>Days</u></th>
                        <th><u>Opening Hours</u></th>
                        <th><u>Bank Holiday Hours</u></th>
                    </tr>
                </thead>
                <tbody>
                    <tr>
                        <td>MONDAY</td>
                        <td>10AM - 11PM</td>
                        <td>12PM - 10PM</td>
                    </tr>
                    <tr>
                        <td>TUESDAY</td>
                        <td>10AM - 11PM</td>
                        <td>12PM - 10PM</td>
                    </tr>
                    <tr>
                        <td>WEDNESDAY</td>
                        <td>10AM - 11PM</td>
                        <td>12PM - 10PM</td>
                    </tr>
                    <tr>
                        <td>THURSDAY</td>
                        <td>10AM - 11PM</td>
                        <td>12PM - 10PM</td>
                    </tr>
                    <tr>
                        <td>FRIDAY</td>
                        <td>10AM - 11PM</td>
                        <td>12PM - 10PM</td>
                    </tr>
                    <tr>
                        <td>SATURDAY</td>
                        <td>10AM - 11PM</td>
                        <td>12PM - 10PM </td>
                    </tr>
                    <tr>
                        <td>SUNDAY</td>
                        <td>12PM - 10PM</td>
                        <td>CLOSED</td>
                    </tr>
                </tbody>
            </Table>

        </div>
    );
}

export default Tables;