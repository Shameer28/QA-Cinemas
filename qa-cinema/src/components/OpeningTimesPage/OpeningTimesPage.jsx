import Tables from "./Tables";
import { Container, Image } from 'react-bootstrap'
import image1 from './../img/openingtimes.png'

const OpeningTimesPage = () => {
    return (
        <div>
            < Image src={image1} alt="About us photo" width="100%" /> <br />

            <Container>
                <h1>QA Cinemas' Opening Hours :</h1><br />
                <Tables />
            </Container>
        </div>
    );
}

export default OpeningTimesPage;