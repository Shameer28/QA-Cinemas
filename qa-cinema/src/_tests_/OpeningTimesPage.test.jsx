import OpeningTimesPage from "../components/OpeningTimesPage/OpeningTimesPage";
import { create } from 'react-test-renderer';

describe('Opening Times Page testing', () => {

    it(`Should match the snapshot`, () => {
        const instance = create(<OpeningTimesPage />).toJSON();
        expect(instance).toMatchSnapshot();
    })
})

