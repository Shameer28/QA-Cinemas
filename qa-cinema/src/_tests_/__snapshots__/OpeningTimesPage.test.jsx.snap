// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Opening Times Page testing Should match the snapshot 1`] = `
<div
  class="background"
>
  <img
    alt="About us photo"
    className=""
    src="openingtimes.png"
    width="100%"
  />
   
  <br />
  <br />
  <div
    className="container"
  >
    <h2
      style={
        Object {
          "color": "white",
          "fontWeight": "400",
          "fontfamily": "isonormregular, sans-serif",
          "letterSpacing": ".15em",
          "lineHeight": "1.1",
          "textTransform": "uppercase",
        }
      }
    >
      QA Cinemas' Opening Hours :
    </h2>
    <br />
    <div>
      <table
        className="table table-dark table-sm table-striped table-bordered table-hover"
        style={
          Object {
            "backgroundColor": "#212529",
            "border": "solid",
            "borderColor": "black",
            "color": "white",
          }
        }
      >
        <thead>
          <tr>
            <th>
              <u>
                Days
              </u>
            </th>
            <th>
              <u>
                Opening Hours
              </u>
            </th>
            <th>
              <u>
                Bank Holiday Hours
              </u>
            </th>
          </tr>
        </thead>
        <tbody>
          <tr>
            <td>
              MONDAY
            </td>
            <td>
              10AM - 11PM
            </td>
            <td>
              12PM - 10PM
            </td>
          </tr>
          <tr>
            <td>
              TUESDAY
            </td>
            <td>
              10AM - 11PM
            </td>
            <td>
              12PM - 10PM
            </td>
          </tr>
          <tr>
            <td>
              WEDNESDAY
            </td>
            <td>
              10AM - 11PM
            </td>
            <td>
              12PM - 10PM
            </td>
          </tr>
          <tr>
            <td>
              THURSDAY
            </td>
            <td>
              10AM - 11PM
            </td>
            <td>
              12PM - 10PM
            </td>
          </tr>
          <tr>
            <td>
              FRIDAY
            </td>
            <td>
              10AM - 11PM
            </td>
            <td>
              12PM - 10PM
            </td>
          </tr>
          <tr>
            <td>
              SATURDAY
            </td>
            <td>
              10AM - 11PM
            </td>
            <td>
              12PM - 10PM 
            </td>
          </tr>
          <tr>
            <td>
              SUNDAY
            </td>
            <td>
              12PM - 10PM
            </td>
            <td>
              CLOSED
            </td>
          </tr>
        </tbody>
      </table>
    </div>
    <br />
  </div>
</div>
`;
